app/src/main/java/at/sphericalk/gidget/ui/routes/Feed.kt

```diff
@@ -3,6 +3,7 @@ package at.sphericalk.gidget.ui.routes
 import android.content.Context
 import android.content.Intent
 import android.net.Uri
+import android.util.Log
```

Comment:
Seems unused?

Reply:
fixed in https://github.com/SphericalKat/gimmick/pull/5/commits/810740dcf36b3b0fdcc36e3dd8869e7deacd0d1e